### PR TITLE
[feature](analyze_cmd) add show-tablets-belong stmt for analyzing a batch of tablet-ids

### PR DESCRIPTION
**Pull request by Yulei-Yang**
Some cmds return tablet-id list, such as: `show proc '/cluster_health/tablet_health/73421';`
image

but it is hard to identify the tables and show their infos, which these tablets belong to.
this pr add a show-tablets-belong stmt to resolve this problem.
image
 

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yulei-Yang** [comment]
run buildall

**Review by xy720 — DISMISSED** [review]
LGTM

File: docs/sidebars.json [hunk 1]
```diff
@@ -867,7 +867,8 @@
                                 "sql-manual/sql-reference/Database-Administration-Statements/RECOVER",
                                 "sql-manual/sql-reference/Database-Administration-Statements/KILL",
                                 "sql-manual/sql-reference/Database-Administration-Statements/ADMIN-REBALANCE-DISK",
-                                "sql-manual/sql-reference/Database-Administration-Statements/ADMIN-CANCEL-REBALANCE-DISK"
+                                "sql-manual/sql-reference/Database-Administration-Statements/ADMIN-CANCEL-REBALANCE-DISK",
+                                "sql-manual/sql-reference/Database-Administration-Statements/UNSET-VARIABLE"
```
Comment (lide-reed) [inline docs/sidebars.json:871]: It seems about the change of unset variable , remove from here?

Comment (Yulei-Yang) [reply docs/sidebars.json:871]: unset variable stmt is brought by me via https://github.com/apache/doris/pull/27552, but I forgot to add its document to sidebar, so fix it here

**Review by lide-reed — DISMISSED** [review]
LGTM

**Review by morrySnow — CHANGES_REQUESTED** [review]
What does it have to do with `ANALYZE`? `ANALYZE` should only used for collection statistic infomation. i cannot find any relation bewteen this statement and `ANALYZE`. If u want show detail info about tablet, please use more appropriate keyword

**Comment by Yulei-Yang** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by Yulei-Yang** [comment]
run feut

**Comment by Yulei-Yang** [comment]
run p0

**Comment by Yulei-Yang** [comment]
run buildall

**Review by xy720 — APPROVED** [review]

**Review by lide-reed — APPROVED** [review]
LGTM

**Pull request merged**